src/Sylius/Behat/Context/Ui/CartContext.php

```diff
@@ -164,4 +179,41 @@ public function thereShouldBeNoDiscount()
 
         expect($this->cartSummaryPage)->toThrow(ElementNotFoundException::class)->during('getPromotionTotal', []);
     }
+
+    /**
+     * @Then /^(its|theirs) price should be decreased by ("[^"]+")$/
+     * @Then /^(product "[^"]+") price should be decreased by ("[^"]+")$/
+     */
+    public function itsPriceShouldBeDecreasedBy(ProductInterface $product, $amount)
+    {
+        $this->cartSummaryPage->open();
+
+        $discountPrice = $this->getPriceFromString($this->cartSummaryPage->getItemDiscountPrice($product->getName()));
+        $regularPrice = $this->getPriceFromString($this->cartSummaryPage->getItemRegularPrice($product->getName()));
+
+        expect($discountPrice)->toBe($regularPrice - $amount);
+    }
+
+    /**
+     * @Given /^(product "[^"]+") price should not be decreased$/
+     */
+    public function productPriceShouldNotBeDecreased(ProductInterface $product)
+    {
+        $this->cartSummaryPage->open();
+
+        expect($this->cartSummaryPage)
+            ->toThrow(ElementNotFoundException::class)
```

Comment:
I don't think we should expect element not found exception here. It should rather be:

```
expect($this->cartSummaryPage->isProductDiscounted($product)->toBe(false);
```